cypress/utils/dashboards/vis_builder/constants.js

```diff
@@ -26,9 +26,15 @@ export const VB_LINE_VIS_TITLE = 'VB: Basic Line Chart';
 export const VB_PATH_FIXTURE = 'dashboard/opensearch_dashboards/visBuilder/';
 export const VB_PATH_INDEX_DATA = VB_PATH_FIXTURE + VB_INDEX_DATA;
 export const VB_PATH_SO_DATA = VB_PATH_FIXTURE + VB_SO_DATA;
+export const VISUALIZE_APP_PATH = '/app/visualize';
```

Comment:
nit: should this go under 

`// App URL Paths` section and the data paths here?